CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ option(OCOS_ENABLE_SELECTED_OPLIST "Enable including the selected_ops tool file"
 option(OCOS_BUILD_PYTHON "Enable building the Python package" OFF)
 option(OCOS_BUILD_JAVA "Enable building the Java package" OFF)
 option(OCOS_BUILD_ANDROID "Enable building the Android package" OFF)
+option(OCOS_BUILD_APPLE_FRAMEWORK "Enable building of the MacOS/iOS framework" OFF)
```

Comment:
Is it 'Apple framework' or 'iOS framework'?

Reply:
I think at this level in CMakeLists.txt it is flexible enough to be either for iOS or MacOS. so went with the more generic "APPLE"

Reply:
I don't know too much macOS framework, was it very similar to iOS's one on building script? If not, the iOS framework seems more specific.

Reply:
`tools/ios/build_xcframework.py` is iOS specific. but here it should work for MacOS as well, at the moment.